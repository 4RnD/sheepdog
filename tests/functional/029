#!/bin/bash

# Test sheep VDI redundancy level

seq=`basename $0`
echo "QA output created by $seq"

here=`pwd`
tmp=/tmp/$$
status=1        # failure is the default!

# get standard environment, filters and checks
. ./common.rc
. ./common.filter

_cleanup

for i in `seq 0 6`; do
	_start_sheep $i;
done

_wait_for_sheep 7

$COLLIE cluster format -c 3

for i in `seq 2 4`; do
	$COLLIE vdi create test$i 20M -c $i -P
done

$COLLIE vdi snapshot -s tag test2
$COLLIE vdi clone -s 1 test2 clone -c 3


for i in `seq 2 3`; do
	_kill_sheep $i;
done

for i in `seq 7 8`; do
	_start_sheep $i;
done

_wait_for_sheep_recovery 0

$COLLIE vdi delete clone
$COLLIE vdi delete -s 1 test2

$COLLIE vdi list | _filter_short_date

for i in `seq 2 4`; do
	$COLLIE vdi object test$i -i 1;
done

