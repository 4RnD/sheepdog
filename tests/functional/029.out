QA output created by 029
using backend plain store
Invalid parameter 16:16
To create replicated vdi, set -c x
  x(1 to 31)   - number of replicated copies
To create erasure coded vdi, set -c x:y
  x(2,4,8,16)  - number of data strips
  y(1 to 15)   - number of parity strips
  Name        Id    Size    Used  Shared    Creation time   VDI id  Copies  Tag
  test5        0   20 MB   20 MB  0.0 MB DATE   fd2c30    4:2              
  test4        0   20 MB   20 MB  0.0 MB DATE   fd2de3      4              
  test7        0   20 MB   20 MB  0.0 MB DATE   fd2f96    4:4              
  test6        0   20 MB   20 MB  0.0 MB DATE   fd3149    4:3              
  test3        0   20 MB   20 MB  0.0 MB DATE   fd3662      3              
  test2        0   20 MB  0.0 MB   20 MB DATE   fd3816      2              
  test9        0   20 MB   20 MB  0.0 MB DATE   fd4094   16:7              
  test8        0   20 MB   20 MB  0.0 MB DATE   fd4247    4:5              
Looking for the object 0xfd381500000001 (vid 0xfd3816 idx 1, 2 copies) with 23 nodes

127.0.0.1:7000 doesn't have the object
127.0.0.1:7001 doesn't have the object
127.0.0.1:7004 doesn't have the object
127.0.0.1:7005 doesn't have the object
127.0.0.1:7006 doesn't have the object
127.0.0.1:7007 doesn't have the object
127.0.0.1:7008 doesn't have the object
127.0.0.1:7009 doesn't have the object
127.0.0.1:7010 doesn't have the object
127.0.0.1:7011 doesn't have the object
127.0.0.1:7012 doesn't have the object
127.0.0.1:7013 doesn't have the object
127.0.0.1:7014 doesn't have the object
127.0.0.1:7015 doesn't have the object
127.0.0.1:7016 has the object
127.0.0.1:7017 doesn't have the object
127.0.0.1:7018 doesn't have the object
127.0.0.1:7019 doesn't have the object
127.0.0.1:7020 doesn't have the object
127.0.0.1:7021 doesn't have the object
127.0.0.1:7022 has the object
127.0.0.1:7023 doesn't have the object
127.0.0.1:7024 doesn't have the object
Looking for the object 0xfd366200000001 (vid 0xfd3662 idx 1, 3 copies) with 23 nodes

127.0.0.1:7000 doesn't have the object
127.0.0.1:7001 has the object
127.0.0.1:7004 doesn't have the object
127.0.0.1:7005 doesn't have the object
127.0.0.1:7006 doesn't have the object
127.0.0.1:7007 has the object
127.0.0.1:7008 doesn't have the object
127.0.0.1:7009 doesn't have the object
127.0.0.1:7010 doesn't have the object
127.0.0.1:7011 doesn't have the object
127.0.0.1:7012 has the object
127.0.0.1:7013 doesn't have the object
127.0.0.1:7014 doesn't have the object
127.0.0.1:7015 doesn't have the object
127.0.0.1:7016 doesn't have the object
127.0.0.1:7017 doesn't have the object
127.0.0.1:7018 doesn't have the object
127.0.0.1:7019 doesn't have the object
127.0.0.1:7020 doesn't have the object
127.0.0.1:7021 doesn't have the object
127.0.0.1:7022 doesn't have the object
127.0.0.1:7023 doesn't have the object
127.0.0.1:7024 doesn't have the object
Looking for the object 0xfd2de300000001 (vid 0xfd2de3 idx 1, 4 copies) with 23 nodes

127.0.0.1:7000 doesn't have the object
127.0.0.1:7001 has the object
127.0.0.1:7004 doesn't have the object
127.0.0.1:7005 has the object
127.0.0.1:7006 doesn't have the object
127.0.0.1:7007 doesn't have the object
127.0.0.1:7008 doesn't have the object
127.0.0.1:7009 doesn't have the object
127.0.0.1:7010 doesn't have the object
127.0.0.1:7011 doesn't have the object
127.0.0.1:7012 doesn't have the object
127.0.0.1:7013 doesn't have the object
127.0.0.1:7014 has the object
127.0.0.1:7015 doesn't have the object
127.0.0.1:7016 doesn't have the object
127.0.0.1:7017 doesn't have the object
127.0.0.1:7018 doesn't have the object
127.0.0.1:7019 has the object
127.0.0.1:7020 doesn't have the object
127.0.0.1:7021 doesn't have the object
127.0.0.1:7022 doesn't have the object
127.0.0.1:7023 doesn't have the object
127.0.0.1:7024 doesn't have the object
Looking for the object 0xfd2c3000000001 (vid 0xfd2c30 idx 1, 6 copies) with 23 nodes

127.0.0.1:7000 has the object
127.0.0.1:7001 has the object
127.0.0.1:7004 doesn't have the object
127.0.0.1:7005 doesn't have the object
127.0.0.1:7006 doesn't have the object
127.0.0.1:7007 doesn't have the object
127.0.0.1:7008 doesn't have the object
127.0.0.1:7009 has the object
127.0.0.1:7010 doesn't have the object
127.0.0.1:7011 has the object
127.0.0.1:7012 doesn't have the object
127.0.0.1:7013 doesn't have the object
127.0.0.1:7014 doesn't have the object
127.0.0.1:7015 doesn't have the object
127.0.0.1:7016 doesn't have the object
127.0.0.1:7017 doesn't have the object
127.0.0.1:7018 doesn't have the object
127.0.0.1:7019 doesn't have the object
127.0.0.1:7020 doesn't have the object
127.0.0.1:7021 has the object
127.0.0.1:7022 doesn't have the object
127.0.0.1:7023 has the object
127.0.0.1:7024 doesn't have the object
Looking for the object 0xfd314900000001 (vid 0xfd3149 idx 1, 7 copies) with 23 nodes

127.0.0.1:7000 doesn't have the object
127.0.0.1:7001 has the object
127.0.0.1:7004 doesn't have the object
127.0.0.1:7005 doesn't have the object
127.0.0.1:7006 doesn't have the object
127.0.0.1:7007 doesn't have the object
127.0.0.1:7008 has the object
127.0.0.1:7009 doesn't have the object
127.0.0.1:7010 doesn't have the object
127.0.0.1:7011 has the object
127.0.0.1:7012 has the object
127.0.0.1:7013 doesn't have the object
127.0.0.1:7014 doesn't have the object
127.0.0.1:7015 doesn't have the object
127.0.0.1:7016 doesn't have the object
127.0.0.1:7017 doesn't have the object
127.0.0.1:7018 doesn't have the object
127.0.0.1:7019 has the object
127.0.0.1:7020 doesn't have the object
127.0.0.1:7021 doesn't have the object
127.0.0.1:7022 has the object
127.0.0.1:7023 doesn't have the object
127.0.0.1:7024 has the object
Looking for the object 0xfd2f9600000001 (vid 0xfd2f96 idx 1, 8 copies) with 23 nodes

127.0.0.1:7000 doesn't have the object
127.0.0.1:7001 has the object
127.0.0.1:7004 has the object
127.0.0.1:7005 doesn't have the object
127.0.0.1:7006 has the object
127.0.0.1:7007 doesn't have the object
127.0.0.1:7008 has the object
127.0.0.1:7009 doesn't have the object
127.0.0.1:7010 doesn't have the object
127.0.0.1:7011 doesn't have the object
127.0.0.1:7012 has the object
127.0.0.1:7013 has the object
127.0.0.1:7014 has the object
127.0.0.1:7015 doesn't have the object
127.0.0.1:7016 doesn't have the object
127.0.0.1:7017 doesn't have the object
127.0.0.1:7018 doesn't have the object
127.0.0.1:7019 has the object
127.0.0.1:7020 doesn't have the object
127.0.0.1:7021 doesn't have the object
127.0.0.1:7022 doesn't have the object
127.0.0.1:7023 doesn't have the object
127.0.0.1:7024 doesn't have the object
Looking for the object 0xfd424700000001 (vid 0xfd4247 idx 1, 9 copies) with 23 nodes

127.0.0.1:7000 doesn't have the object
127.0.0.1:7001 doesn't have the object
127.0.0.1:7004 has the object
127.0.0.1:7005 doesn't have the object
127.0.0.1:7006 has the object
127.0.0.1:7007 doesn't have the object
127.0.0.1:7008 doesn't have the object
127.0.0.1:7009 doesn't have the object
127.0.0.1:7010 has the object
127.0.0.1:7011 has the object
127.0.0.1:7012 doesn't have the object
127.0.0.1:7013 doesn't have the object
127.0.0.1:7014 has the object
127.0.0.1:7015 has the object
127.0.0.1:7016 doesn't have the object
127.0.0.1:7017 doesn't have the object
127.0.0.1:7018 doesn't have the object
127.0.0.1:7019 doesn't have the object
127.0.0.1:7020 doesn't have the object
127.0.0.1:7021 doesn't have the object
127.0.0.1:7022 has the object
127.0.0.1:7023 has the object
127.0.0.1:7024 has the object
Looking for the object 0xfd409400000001 (vid 0xfd4094 idx 1, 23 copies) with 23 nodes

127.0.0.1:7000 has the object
127.0.0.1:7001 has the object
127.0.0.1:7004 has the object
127.0.0.1:7005 has the object
127.0.0.1:7006 has the object
127.0.0.1:7007 has the object
127.0.0.1:7008 has the object
127.0.0.1:7009 has the object
127.0.0.1:7010 has the object
127.0.0.1:7011 has the object
127.0.0.1:7012 has the object
127.0.0.1:7013 has the object
127.0.0.1:7014 has the object
127.0.0.1:7015 has the object
127.0.0.1:7016 has the object
127.0.0.1:7017 has the object
127.0.0.1:7018 has the object
127.0.0.1:7019 has the object
127.0.0.1:7020 has the object
127.0.0.1:7021 has the object
127.0.0.1:7022 has the object
127.0.0.1:7023 has the object
127.0.0.1:7024 has the object
